### PR TITLE
Test cases passed

### PR DESCRIPTION
**Pull request by vaibhavpacharya**
1. Cucumber test cases, feature.rb, stepdefinition.rb for homepage.

### CODE REVIEW

File: features/homepage.feature [hunk 1]
```diff
@@ -5,37 +5,30 @@ Feature: Homepage for Bing Bites
...
-   Then I should see the "BingBites" homepage
+   When I click on the link
+   Then I should see the Welcome message
 
 
 Scenario: Product description
    Given I am on the home page
-   And I press the "product desc" 
-   Then I should see the "product info"
+   When I click on the product desc
```
Comment (suyogrnerkar) [inline features/homepage.feature]: I click on ... not needed..no link or button to click

File: features/homepage.feature [hunk 2]
```diff
@@ -5,37 +5,30 @@ Feature: Homepage for Bing Bites
 
 Scenario: Go to HomePage
    Given I am on the home page
-   And I click on the "BingBites"
-   Then I should see the "BingBites" homepage
+   When I click on the link
+   Then I should see the Welcome message
```
Comment (suyogrnerkar) [inline features/homepage.feature]: When I click on the link ??? Need it ?

File: features/homepage.feature [hunk 3]
```diff
@@ -5,37 +5,30 @@ Feature: Homepage for Bing Bites
...
-   And I press the "product desc" 
-   Then I should see the "product info"
+   When I click on the product desc
+   Then I should see the product info
 
 Scenario: Team members
    Given I am on the home page
-   And I press the "team members"
-   Then I should see the "team members"
+   When I click on the "team members"
```
Comment (suyogrnerkar) [inline features/homepage.feature]: click on ??? where is the link ?

File: features/homepage.feature [hunk 4]
```diff
@@ -5,37 +5,30 @@ Feature: Homepage for Bing Bites
...
+   When I click on the single sign up
+   Then I should goto homepage
 
 
+Scenario: Visiting the sign up page
+    Given I am on the home page
+    Then I should see the sign up button
+    
+Scenario: Visiting the sign in page
+    Given I am on the home page
```
Comment (suyogrnerkar) [inline features/homepage.feature]: indentation problem

File: features/step_definitions/stepdefinition.rb [hunk 5]
```diff
@@ -2,11 +2,52 @@
...
 
-When(/^I click on the "([^"]*)" button to Login/) do |button|
-  click_button "Sign In"
-end
+    Then(/^I should see the Welcome message$/) do
+    expect(page).to have_content("Welcome to BingBites")
+    end
+
+    When(/^I click on the product desc$/) do
+    visit root_path
```
Comment (suyogrnerkar) [inline features/step_definitions/stepdefinition.rb]: this should have logic like something which actually clicks

File: features/step_definitions/stepdefinition.rb [hunk 6]
```diff
@@ -2,11 +2,52 @@
...
+    end
+    
+    Then (/^I should see the product info$/) do
+    expect(page).to have_content("amazing experience")
+    end
+    
+    When(/^I click on the "team members"$/) do
+    visit root_path
```
Comment (suyogrnerkar) [inline features/step_definitions/stepdefinition.rb]: refer https://github.com/BingBites/bingbites/pull/20/files#r107822548

File: features/step_definitions/stepdefinition.rb [hunk 7]
```diff
@@ -2,11 +2,52 @@
...
+    end
+    
+    Then (/I should see the team members$/) do
+    expect(page).to have_content("The Team")
+      end
+    
+    When(/^I click on the single sign up$/) do
+    visit root_path
```
Comment (suyogrnerkar) [inline features/step_definitions/stepdefinition.rb]: same as https://github.com/BingBites/bingbites/pull/20/files#r107822548

File: features/step_definitions/stepdefinition.rb [hunk 8]
```diff
@@ -2,11 +2,52 @@
...
+
+    When(/^I click on the product desc$/) do
+    visit root_path
+    end
+    
+    Then (/^I should see the product info$/) do
+    expect(page).to have_content("amazing experience")
+    end
+    
+    When(/^I click on the "team members"$/) do
```
Comment (suyogrnerkar) [inline features/step_definitions/stepdefinition.rb]: there's nothing to click for team members

File: features/step_definitions/stepdefinition.rb [hunk 9]
```diff
@@ -2,11 +2,52 @@
...
+    end
 
-When(/^I click on the "([^"]*)" button to Login/) do |button|
-  click_button "Sign In"
-end
+    Then(/^I should see the Welcome message$/) do
+    expect(page).to have_content("Welcome to BingBites")
+    end
+
+    When(/^I click on the product desc$/) do
```
Comment (suyogrnerkar) [inline features/step_definitions/stepdefinition.rb]: nothing to click

File: features/step_definitions/stepdefinition.rb [hunk 10]
```diff
@@ -2,11 +2,52 @@
 require 'cgi'
 require File.expand_path(File.join(File.dirname(__FILE__), "..", "support", "paths"))
 
-When(/^I click on the "([^"]*)" button to sign up$/) do |button|
-  click_button "Sign Up"
-end
+    When(/^I click on the link$/) do
```
Comment (suyogrnerkar) [inline features/step_definitions/stepdefinition.rb]: link should be specifically named as bingbites

File: features/homepage.feature (hunk 3 above)
Comment (dmajumd2) [reply features/homepage.feature]: click on is removed

File: features/step_definitions/stepdefinition.rb (hunk 9 above)
Comment (dmajumd2) [reply features/step_definitions/stepdefinition.rb]: Done, nothing to click on product description

**Pull request merged**